plugin/trino-postgresql/src/test/resources/postgresql.conf

```diff
@@ -0,0 +1,630 @@
+# -----------------------------
```

Comment:
What is this file for?

Reply:
> What is this file for?

To configure PostgreSQL server.

> The file comes from XXXX

This is default configuration file for PostgreSQL. All configuration is commented. It nicely self documented, but also it is very verbose so I have mixed feelings if it makes sense to keep it here.